src/main/java/edu/asu/plp/tool/backend/isa/events/DeviceOutputEvent.java

```diff
@@ -1,5 +1,10 @@
 package edu.asu.plp.tool.backend.isa.events;
 
+/**
```

Comment:
Change the class description as I modified in DeviceInputEvent.java, but in a reverse way. (back-end to front-end).